cves/CVE-2012-2779.yml

```diff
@@ -360,11 +394,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 20
 CWE_note:
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it. If the
   report mentions a nickname, e.g. "Heartbleed", use that. Or come up with one!
 
   Must be under 30 characters. Optional. Be appropriate.
-nickname:
+nickname: Know Thy Input
```

Comment:
Going to pause my review here - you're on the right track. Circle back and see if you can learn more about the format and context. Some more observations would be nice.